src/mem/old_api.cc

```diff
@@ -329,7 +198,7 @@ memAllocBuf(size_t net_size, size_t * gross_size)
     else {
         ++HugeBufCountMeter;
         HugeBufVolumeMeter += *gross_size;
```

Comment:
This PR makes gross_size parameter optional in src/mem/minimal.cc and src/tests/stub_libmem.cc, but the parameter is still required in this (primary) memAllocBuf() implementation. This PR also adds a memAllocBuf() caller that supplies nil gross_size and can crash this code.

Please adjust to keep all memAllocBuf() implementations in sync. For example, we can make the parameter optional in all memAllocBuf() implementations, including this one:

```suggestion
        HugeBufVolumeMeter += net_size;
```